CHANGELOG.md

```diff
@@ -1065,6 +1065,7 @@ Released 2018-09-13
 [`modulo_one`]: https://rust-lang.github.io/rust-clippy/master/index.html#modulo_one
 [`multiple_crate_versions`]: https://rust-lang.github.io/rust-clippy/master/index.html#multiple_crate_versions
 [`multiple_inherent_impl`]: https://rust-lang.github.io/rust-clippy/master/index.html#multiple_inherent_impl
+[`must_use_unit`]: https://rust-lang.github.io/rust-clippy/master/index.html#must_use_unit
```

Comment:
Should this maybe be called `needless_must_use`? It would be more in line with our other `needless_*` lints.

Reply:
I'm not married to the name, but I'd expect a `needless_must_use` lint to check for more than the a unit result. On the other hand, I cannot think of anything else to check, so maybe that's a moot point.

Reply:
I now have two lints, one is called `must_use_unit` and the other is called `double_must_use`.